tests/command_line/test_bader_caller.py

```diff
@@ -74,7 +74,7 @@ def test_from_path(self):
         chgref_path = f"{direct_dir}/_CHGCAR_sum.gz"
 
         analysis = BaderAnalysis(chgcar_filename=chgcar_path, chgref_filename=chgref_path)
-        analysis_from_path = BaderAnalysis.from_path(from_path_dir)
+        analysis_from_path = analysis.from_path(from_path_dir)
```

Comment:
Wondering how this was passing before... I'm pretty confused 😕 .

Reply:
`from_path` is a `classmethod`. i.e. it's not using any instance attributes so your change shouldn't make any difference. i think the code was clearer before though that the arguments of `BaderAnalysis(chgcar_filename=chgcar_path, chgref_filename=chgref_path)` were not used so i would revert

https://github.com/materialsproject/pymatgen/blob/d871fdfacbb4b08f200bdb64e76b0aaefccaf690/pymatgen/command_line/bader_caller.py#L398-L400

Reply:
Make sense. Sorry I didn't look into the `from_path` careful enough. I guess I changed this because I got an error somehow and cannot recreate it now. Would revert this. Thanks.